rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -137,6 +137,13 @@ enum RocalDecoderType
     ROCAL_DECODER_VIDEO_FFMPEG_HW = 4
 };
 
+enum RocalOutputMemType
+{
+    ROCAL_MEMCPY_HOST = 0,
```

Comment:
Is the defines in 148 to 150 replicating the same functionality? Please remove the one which is not used

Reply:
Those defines are used in getImageLabels API and similar metadata API functions to force return the metadata in GPU memory if set. I tried using the already present defines but I wasn't able to add python bindings for defines so I added an enum.